app/views/merchant/discounts/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>New Discount</h2>
+<%= form_tag "/merchant/discounts" do %>
+  <%= label_tag :percent_discount %>
+  <%= text_field_tag :percent_discount %>
+  <%= label_tag :minimum_quantity %>
+  <%= text_field_tag :minimum_quantity %>
+  <%= label_tag :minimum_value %>
+  <%= text_field_tag :minimum_value %>
+  <%= label_tag :applicable_items %>
+
+  <%= collection_check_boxes(:applicable_items, :id, @items, :id, :name) do |item| %>
```

Comment:
This is functional, and better than the checkboxes I created for pet_application in the paired project which could only work with one checkbox selected, but it is still not perfect. It results in an array of the item id's passed with the params as `id => []` but always includes an empty string as the first item in the array. I am able to delete this in the controller before creating discount_items, but then that creates an array within the array and have to call `.flatten`. There must be a more elegant solution, but for now I'm just happy it works and definitely an improvement.